client/components/repository/common/CreateDialog/index.vue

```diff
@@ -93,6 +100,7 @@ export default {
       this.$emit('created', item);
       this.visible = false;
       this.$router.push({ query: { activityId: item.id } });
+      this.submitting = false;
```

Comment:
Should this go straight after the event is emitted to make it more contextualised 🤔